### PR TITLE
[PUBLISHED] 89 - update readmes

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: 2-structured-data/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Getting started with PHP on GCP
+# 1 - Hello world
```
Comment (JustinBeckwith) [inline 2-structured-data/README.md]: should be step 2 :)

File: 5-logging/README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-# Getting started with PHP on GCP
+# 4 - Authenticating users
```
Comment (JustinBeckwith) [inline 5-logging/README.md]: should be step 5

**Comment by bshaffer** [comment]
@JustinBeckwith fixed! PTAL

**Comment by JustinBeckwith** [comment]
LGTM

**Pull request merged**